proto/kava/auction/v1beta1/query.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package kava.auction.v1beta1;
+
+import "kava/auction/v1beta1/genesis.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/kava-labs/kava/x/auction/types";
+
+// Query defines the gRPC querier service for auction module
+service Query {
+  // Params queries all parameters of the auction module.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/auction/params";
```

Comment:
```suggestion
    option (google.api.http).get = "/kava/auction/v1beta1/params";
```
same for other endpoints

Reply:
https://github.com/Kava-Labs/kava/pull/1052/commits/e3c0f253bd10ae79005926603c17e065303e1eb3